crates/driver/src/infra/notify/notification.rs

```diff
@@ -51,6 +51,10 @@ pub enum Kind {
 pub enum ScoreKind {
     /// No clearing prices are present for all trades.
     InvalidClearingPrices,
+    /// Trade: invalid executed amount
+    InvalidExecutedAmount,
+    /// missing native price for token {0:?}
```

Comment:
These doc comments could be more explicit. Also why use the debug notation here?